src/frontend/src/lib/components/ui/Json.svelte

```diff
@@ -21,7 +22,7 @@
 		| 'undefined';
 
 	const getValueType = (value: unknown): ValueType => {
-		if (value === null) {
+		if (isNullish(value)) {
```

Comment:
I think we want to display `undefined` here and the `=== null` is a choice

Reply:
right! I am putting the eslint exception here